.github/workflows/deploy-next-js-dev.yml

```diff
@@ -0,0 +1,111 @@
+on:
+  workflow_call:
+    inputs:
+      NODE_VERSION:
+        required: false
+        type: string
+        default: "20"
+      NAIS_VARS:
+        required: false
+        type: string
+        default: ".nais/dev.yml"
+      SKIP_DRAFT_RELEASE:
+        required: false
+        type: boolean
+        default: false
+    outputs:
+      image:
+        description: "Image from nais build push action"
+        value: ${{ jobs.deploy-dev.outputs.image }}
+jobs:
+  deploy-dev:
+    permissions:
+      contents: write
+      id-token: write
+      packages: read
+      security-events: write
+      actions: read
+    runs-on: ubuntu-latest
+    outputs:
+      image: ${{ steps.docker-build-push.outputs.image }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ inputs.NODE_VERSION }}
+          cache: 'npm'
+      - name: next.js cache
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.npm
+            ${{ github.workspace }}/.next/cache
+          key: ${{ runner.os }}-nextjs-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('**/*.js', '**/*.jsx', '**/*.ts', '**/*.tsx') }}
+          restore-keys: |
+            ${{ runner.os }}-nextjs-${{ hashFiles('**/package-lock.json') }}-
+      - name: Install npm dependencies
+        env:
+          NPM_AUTH_TOKEN: ${{ secrets.READER_TOKEN }}
+        run: npm ci
+      - name: Pre-deploy
+        uses: navikt/pam-deploy/actions/pre-deploy@v7
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PROJECT_ID: ${{ vars.NAIS_MANAGEMENT_PROJECT_ID }}
+          TEAM: "teampam"
```

Comment:
Ta inn team som optional defaulted variabel, vi har flere teams som bruker pam-deploy (bla digihot), og det er fint om de kan fortsette å gjøre det selv for nye ting.

Reply:
Fixat!